openvpn_exporter.go

```diff
@@ -52,8 +53,12 @@ var (
 		[]string{"status_path"}, nil)
 
 	// Metrics specific to OpenVPN servers.
+	openvpnConnectedClientsDesc = prometheus.NewDesc(
+		prometheus.BuildFQName("openvpn", "", "connected_clients"),
+		"Number Of Connected Clients", nil, nil)
```

Comment:
Is there any need for having this as a metric explicitly? One could just use `count()` for this, right?

Reply:
This is actually the only metric I use. I thought it would be nice to have it directly.